teamsyncd/teamsync.h

```diff
@@ -11,19 +11,23 @@
 #include "netmsg.h"
 #include <team.h>
 
+// seconds
+const uint32_t DEFAULT_WR_PENDING_TIMEOUT = 70;
```

Comment:
Would like to understand the logic behind this default value of 70 seconds here.

With lacp slow mode, 30 second interval * 3 times = 90 seconds,  all the teamd related processing has to be within that range from teamd shutdown to the lacp connection reestablished.   
During warm reboot, are you expecting it will take about 20 seconds from teamd shutdown to teamd restarted,  then from teamd started to lacp connected, it will take x*10 seconds?